doc/migrations/4.2.0_5.0.0.md

```diff
@@ -3,21 +3,22 @@
 The migrations scripts for Postgres, MySQL, MSSQL can be found in the [`priv/migrations`](../../priv/migrations/) directory. Please remember to provide the existing server domain for the `server` column instead of the localhost.
 
 ### Changes in XEPs:
-- `mod_last` 
-   - Table `last` - added server column, updated primary key and indexes.
+
```

Comment:
Note: the link to `priv/migrations` does note work as it references a file in the repo outside of the docs directory. Maybe replace with a GitHub link?

Reply:
I think so. This is the similar case to linking the documentation from README.md. These links will work in GitHub, but a full link should not cause any problems.